templates/projects/non_stats.html

```diff
@@ -14,9 +14,12 @@
         <span class="alert alert-warning"><strong>Ooops</strong> Sorry, the project does not have tasks to process and
             show some statistics</span>
         </div>
-        <div class="row" style="margin-top: 2em;" id="grafana-link">
-            <a class="btn btn-primary" target="_blank" href="https://grafana.prod.bloomberg.com/d/f59623f5-0ab2-4849-80d3-f4ef8bda7259/poc-crowd-task-metrics?var-project_id={{project.id}}&var-task_source=GIGWORK_{{'PRIV' if private_instance else 'PUBLIC'}}">Grafana Metrics Dashboard</a>
-        </div>
+        {% if config.get('STATS_DASHBOARD_URL') %}
+            {% set dashboard_url = config.get('STATS_DASHBOARD_URL') + "?var-project_id=" + project.id|string + "&var-task_source=GIGWORK_" + ("PRIV" if private_instance else "PUBLIC") %}
+            <div class="row" style="margin-top: 2em;" id="grafana-link">
+                <a class="btn btn-primary" target="_blank" href="{{dashboard_url}}">Grafana Metrics Dashboard</a>
```

Comment:
Could we change the label to "Metrics Dashboard" to be more generic with all types of projects? Thank you .